src/components/ebay-tabs/component.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
             const keyCode = event.charCode || event.keyCode;
             const direction = keyCode === 37 || keyCode === 38 ? -1 : 1;
             const index = (state.index + len + direction) % len;
-            this.getEl(`tab-${index}`).focus();
+            this.getEl(`tabs-${index}`).focus();
```

Comment:
I feel like renaming the `key` here is not necessary.

Reply:
The key has changed too from tab to tabs. 
As per Ian, we should change names to be consistent.